integration-tests/steps/stack_groups.py

```diff
@@ -204,13 +204,6 @@ def step_impl(context, stack_group_name, status):
         assert response in expected_response
 
 
-@then("no resources are described")
```

Comment:
there are checks in this file for tests in `integration-tests/features/describe-stack-group-resources.feature`.  are those needed anymore? if not can we remove them?

Reply:
Good pickups! Another 154 lines could be deleted.
```
% git diff --cached --stat 
 integration-tests/steps/stack_groups.py | 115 -------------------------------------------------------------------------------------------------------------------
 integration-tests/steps/stacks.py       |  39 ---------------------------------------
 2 files changed, 154 deletions(-)
```